src/index.ts

```diff
@@ -2,3 +2,4 @@ export * as auth from "./auth";
 export * as types from "./types";
 export * as webln from "./webln";
 export { Client } from "./client";
+export * from "./decodeInvoice";
```

Comment:
hmmm, I don't think this is a nice API. 
Just exposing the global function because that people don't have to import it from the light-bolt11-decoder. 

doesn't this make more sense to have in the `Invoice` https://github.com/getAlby/js-lightning-tools/blob/master/src/invoice.ts

So I can initialize an invoice and then have all the data available there?

Reply:
@bumi sounds good to me. Would it make sense for this sdk to depend on lightning-tools? or should people install both packages as they are doing now? I think it would be nice to have some support for decoding invoices in this library?

Reply:
Update: we could add a small link to the README

# Decoding Invoices
See [lightning-tools](https://github.com/getAlby/js-lightning-tools)